### ISSUE TITLE
[Bug]: sideBySide does not read properly with a screen reader

### ISSUE THREAD

**Issue by virginiaMae** [Doenet/DoenetML#1456]
### Your DoenetML

```xml
<section boxed><title>use a <tag>sideBySide</tag> with <tag>stack</tag></title>
  <sideBySide margins="5%">
    <div>
      <p><alert>Solution:</alert> Draw a triangle in Quadrant II with the ratio of hypotenuse over opposite equal to <m>5</m> over <m>2</m>, then solve for the adjacent side using the Pythagorean theorem. Choose the negative square root since we are going left on the <m>x</m>-axis. Then we can just read off the ratios between the sides.</p>
    </div>
    <graph identicalAxisScales fixed displayXAxisTickLabels="false" displayYAxis="false" displayXAxis="false" displayYAxisTickLabels="false" xmin="-2.5" xmax=".25" ymin="-.5" ymax="1.25" description="A right triangle in quadrant 2 with opposite side 2 and hypotenuse 5. The adjacent side is thus the negative square root of 21.">
        <polygon vertices="(0,0) (-2,0) (-2,1)"/>
        <polyline vertices="(-1.9,0) (-1.9,0.1) (-2,0.1)"/>
        <label anchor="(-1,.75)"><m>5</m></label>
        <label anchor="(-2.25,.5)"><m>2</m></label>
        <label anchor="(-1.25,-.25)"><m>-\sqrt{21}</m></label>
    </graph>
  </sideBySide>
  <p>Here is a sentence in between the two sideBySide components</p>
  <sideBySide margins="5%">
    <stack>
      <p><m>\sin(\theta) = \dfrac{2}{5}</m></p>
      <p><m>\cos(\theta) = \dfrac{-\sqrt{21}}{5}</m></p>
      <p><m>\tan(\theta) = \dfrac{2}{-\sqrt{21}}</m></p>
    </stack>
    <stack>
      <p><m>\csc(\theta) = \dfrac{5}{2}</m></p>
      <p><m>\sec(\theta) = \dfrac{5}{-\sqrt{21}}</m></p>
      <p><m>\cot(\theta) = \dfrac{-\sqrt{21}}{2}</m></p>
    </stack>
  </sideBySide>
</section>
```

### What happened?

When the screen reader (VoiceOver was tested) reads the math formulas, it skips over cos(theta) and tan(theta) completely. 
It starts with the first formula in the left stack, then goes to the right stack, then reads down the right stack. 

### What version of DoenetML are you using?

v 0.7.20

### What browsers are you seeing the problem on?

Chrome
